Source/AzureSqlBulkFailover.md

```diff
@@ -0,0 +1,191 @@
+# AzureSqlBulkFailover.ps1
+
+This script is used to failover all databases and elastic pools in a subscription to a secondary, already upgraded replica. It filters by subscription, resource group and logical server name. It is intended to be used to facilitate CMW customers to upgrade their databases on demand when upgrades are ready (one touch).
+
+## Usage
+
+This script is intended to be run as an Azure Automation Runbook or locally.
+
+## Parameters
+
+- `SubscriptionId`: The ID of the subscription to failover. Optional.
+- `ResourceGroupName`: The name of the resource group to failover. Optional.
+- `LogicalServerName`: The name of the logical server to failover. Optional.
+
+## Notes
+
+- This script will failover ALL resources that the caller has access to in all subscriptions in the tenant, filtering by SubscriptionId, ResourceGroupName and LogicalServerName.

```

Comment:
"Caller" should be clarified. Maybe "that the service principal that executes the runbook has access to"?